client/src/test/java/com/flipkart/flux/client/runtime/LocalContextTest.java

```diff
@@ -43,23 +43,29 @@ public void setUp() throws Exception {
 
     @Test
     public void testRegisterNew_shouldCreateALocalRegistration() throws Exception {
-        localContext.registerNew("fooBar", 1, "someDescription","someContextId");
-        assertThat(tlStateMachineDef.get()).isEqualTo(new StateMachineDefinition("someDescription", "fooBar", 1l, new HashSet<>(), new HashSet<>(), "someContextId"));
+        localContext.registerNew("fooBar", 1, "someDescription",
```

Comment:
As discussed earlier, declare this constant globally in client Module, on another note are we allowing null for clientElb ?

Reply:
I have added it's default value in FluxClientConfiguration. For other usages, it is being used
via config files wherever possible.
No, we don't allow NULL value for ClientElbId in StateMachines table, if not specified it takes
'defaultElbId' as default value.